cmd/broker/main.go

```diff
@@ -236,7 +236,7 @@ func main() {
 	}
 
 	cfg.OrchestrationConfig.KubernetesVersion = cfg.Provisioner.KubernetesVersion
-
+	cfg.Broker.UpdateCustomResouresLabelsOnAccountMove = true //temp
```

Comment:
Is this for testing only?

Reply:
Why don't you test it via configuration file without hard coded temporary values?